tasks/install-certbot.yml

```
@@ -0,0 +1,14 @@
+---
+- name: Install Certbot from repository and related packages
+  package:
+    name: "{{ item }}"
+    state: "present"
+  loop: "{{ certbot_packages }}"
+
+- name: Install Certbot plugin
+  pip:
+    name: "{{ item }}"
+  loop: "{{ certbot_plugins }}"
+
+- name: Install cron
+  apt: name=cron state=present
```

```suggestion
- name: Install cron
  package:
    name: cron
    state: present
```

Non OS specific task